docs/latest/user/udp-routing.md

```diff
@@ -0,0 +1,59 @@
+# UDP Routing
+
+The [UDPRoute][] resource allows users to configure UDP routing by matching UDP traffic and forwarding it to
+Kubernetes backends. To learn more about UDP routing, refer to the [Gateway API documentation][].
+
+Follow the steps from the [Quickstart Guide](quickstart.md) to install Envoy Gateway and then install the example
+resources used for this guide.
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/latest/examples/kubernetes/udp-routing.yaml
+```
+
+The manifest installs a [GatewayClass][], [Gateway][], UDPRoute and a CoreDNS deployment and service for testing. The 
```

Comment:
The GC and GW from the quick start should be used. Patch the GW as needed using kubectl. See the secure gateways doc as a reference.